myuw/static/css/cards.less

```diff
@@ -268,8 +268,15 @@
         a:hover { background-color: #ddd; }
     }
 
-} /*** END myuw-cards ***/
+    .myuw-card-subheader {
```

Comment:
@DannyNineNineNine not sure we really need a new style here for this. Could we not just use the `myuw-card-heading` existing class?

Reply:
@klpittman: that was done by @ezturner. There is minimal visual differences (size, padding), so unless there's a reason Ethan can explain, I have no objections to switching the class and having a new one.

Reply:
Ah ok. Let me know if you need help making these changes.